Makefile

```diff
@@ -103,14 +102,6 @@ rpm-in-mock-f29: srpm
 rpm-in-mock-rawhide: srpm
 	mock --rebuild -r fedora-rawhide-x86_64 ./*.src.rpm
 
-rpm-in-mock-el7: srpm
```

Comment:
@user-cont/the-team is everyone okay with this? that we will drop conu from centos 7? do we actually have some centos users?

Reply:
I guess, python2 is mandatory on centos7 otherwise, we have to python3 from SCL org team, isn't it?

Reply:
There is still an option to install it from PyPI. +1 for removing this